packages/compass-assistant/src/components/assistant-chat.tsx

```diff
@@ -357,6 +361,10 @@ export const AssistantChat: React.FunctionComponent<AssistantChatProps> = ({
     [ensureOptInAndSend, setMessages, track]
   );
 
+  const visibleMessages = messages.filter(
+    (message) => !message.metadata?.isSystemContext
```

Comment:
I'm wondering if we should just have one that's like `isHidden` so it's more reusable